usr/src/man/man1m/zfs.1m

```diff
@@ -754,8 +759,11 @@ for volumes is 8 Kbytes. Any power of 2 from 512 bytes to 128 Kbytes is valid.
 This property can also be referred to by its shortened column name,
 .Sy volblock .
 .It Sy written
-The amount of
+The amount of space
 .Sy referenced
+by this dataset, that was written since the previous snapshot
+.Po i.e. that is not referenced by the previous snapshot
```

Comment:
oh, so that'll work even with the "." that should go outside of the closing parens? I thought about doing that, but assumed I wouldn't be able to put the period outside of the closing parenthesis. I'll try that out.